### ISSUE TITLE
Migrating to .NET Core 3.0 WPF

### ISSUE THREAD

**Issue by slango0513** [trigger-death/WinDirStat.Net#4]
Nice work! Are you considering moving the project to .NET Core 3.0 WPF?

**Comment by trigger-segfault** [comment]
Definitely! It's nice that the WPF designer is finally working for .NET Core 3.0. The switch probably won't be in the immediate future, just to ensure most of the kinks have been worked out, and to avoid any new headaches...* with WPF being the way it is.*

A good solution for now may be to maintain a .NET Core project alongside the current .NET Framework project in the solution. This should at least help compare any important differences between the two and how they function, and allow fallback to .NET Framework if things go wrong.

In addition: It's my understanding that the new `Microsoft.Xaml.Behaviors.Wpf` library (that replaces `Microsoft.Expression.Interactions` & `System.Windows.Interactivity`) does not support .NET Core WPF in some form or other at the moment. But the *support* that was referred to may not be anything relevant.
